src/Microsoft.AspNet.Cors.Core/CorsOptions.cs

```diff
@@ -13,8 +13,30 @@ namespace Microsoft.AspNet.Cors.Core
     /// </summary>
     public class CorsOptions
     {
+        private string _defaultPolicyName = "__DefaultCorsPolicy";
         private IDictionary<string, CorsPolicy> PolicyMap { get; } = new Dictionary<string, CorsPolicy>();
 
+        public string DefaultPolicyName
+        {
+            get
+            {
+                return _defaultPolicyName;
+            }
+
+            set
+            {
+                if (value == null)
+                {
+                    throw new ArgumentNullException("value",
+                                                    Resources.FormatPropertyOfTypeCannotBeNull(
```

Comment:
I am fine removing it, but we do it at other places today. (and I think i got a comment previously which was the opposite guideline) eg. https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.Core/AntiForgery/AntiForgeryOptions.cs#L41-L44